projects/cdk/abstract/control.ts

```diff
@@ -215,6 +215,8 @@ export abstract class AbstractTuiControl<T>
      */
     protected updateValue(value: T): void {
         if (this.disabled || this.valueIdenticalComparator(this.value, value)) {
+            this.checkControlUpdate();
```

Comment:
Why do we need it? 🤔

Reply:
`this.valueIdenticalComparator(this.value, value)` returned `true` when if the value does not match the mask
and therefore we need to update view

Reply:
This would sync native input value to control value with applied mask?